### ISSUE TITLE
ebay-Icon-button: aria-label is missing.

### ISSUE THREAD

**Issue by saiponnada** [eBay/ebayui-core#2190]
# Bug Report

## Description

`aria-label` is missing on storybook even though value of `menu` is set.
https://opensource.ebay.com/ebayui-core/?path=/story/buttons-ebay-icon-button--default

## Screenshots
<img width="1580" alt="image" src="https://github.com/eBay/ebayui-core/assets/6342519/d0b91849-f843-41ff-948a-a48408aa941f">
